server/eclipse-project/src/main/java/us/freeandfair/corla/Main.java

```diff
@@ -103,7 +106,7 @@
    * The logger.
    */
   public static final Logger LOGGER = LogManager.getLogger(LOGGER_NAME);
- 
+  
```

Comment:
Nit: stray trailing whitespace. In Emacs, this can be prevented with: `(add-hook 'before-save-hook 'delete-trailing-whitespace)`. Not sure what the options are in Eclipse.